<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_salles</class>
 <widget class="QDialog" name="gestion_salles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_salles</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/caca.jpg);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>731</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
*{
background:#aaaa7f;
font-family:century gothic;
font-size:15px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#ajouter_chaise
{
color:black;
border-radius:60px;
background:#a50000;
}
#ajouter_chaise:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_supprimer_chaise
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton_supprimer_chaise:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_modifier
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton_modifier:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_4
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_4:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}
#pushButton_5
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_5:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}
#pushButton_6
{
color:black;
border-radius:60px;
background:#99007f;
}
#pushButton_6:hover
{
color:white;
background:#ff5500;
border-radius:60px;
}</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AJOUTER</string>
    </attribute>
    <widget class="QLabel" name="label_nom_film">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TAILLE_CHAISE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_taille_chaise">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_mark_chaise">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MARK_CHAISE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_language_film">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>QUALITE_CHAISE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chaise">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_qualite_chaise">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_chaise">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
*{
background:#aaaa7f;
font-family:century gothic;
font-size:15px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#ajouter_chaise
{
color:black;
border-radius:60px;
background:#a50000;
}
#ajouter_chaise:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_supprimer_chaise
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton_supprimer_chaise:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton:hover
{
color:white;
background:#000000;
border-radius:60px;
}
#pushButton_modifier
{
color:black;
border-radius:60px;
background:#a50000;
}
#pushButton_modifier:hover
{
color:white;
background:#000000;
border-radius:60px;
}
</string>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_id_film">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_CHIASE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_chaise">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>SUPPRIMER</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_CHIASE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_supp_chaise">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer_chaise">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SUPPRIMER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>AFFICHER</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>491</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TRI PAR:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>MODIFIER</string>
    </attribute>
    <widget class="QLabel" name="label_nom_film_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVELLE_TAILLE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mark_chaise_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVELLE_MARK:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_language_film_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NOUVELLE_QUALITE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_id_film_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID_CHIASE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouvelle_taille">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouvelle_mark">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nouvelle_qualite">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>CHERCHER</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cherrcher par ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font-family:century gothic;
font-size:24px;

}
QGroupBox
{
background:#333;
background-image:url(:/caca.jpg);
}
#pushButton_3
{
color:black;
background:#a50000;
border-radius:60px;
}
#pushButton_3:hover
{
color:white;
border-radius:60px;
background:#000000;
}</string>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
